main.rb

```diff
@@ -12,145 +17,60 @@ def initialize
 
   def option_chosen(option)
     case option
-    when '1'
-      all_books
-    when '2'
-      all_people
-    when '3'
-      make_person
-    when '4'
-      make_book
-    when '5'
-      rent_book
-    when '6'
-      list_rents
-    else
-      wrong_option
+    when '1' then all_books
+    when '2' then all_people
+    when '3' then make_person
+    when '4' then make_book
+    when '5' then rent_book
+    when '6' then PrintRental.new.list_rents(@people)
+    else WrongOption.new.print_message
     end
   end
 
-  def print_book(book, index = nil)
-    "#{index + 1}) #{book.title} by #{book.author}"
-  end
-
   def all_books
     puts 'Books List: '
-    @books.each_with_index { |book, index| puts print_book(book, index) }
-    return 'No books' if @books.empty?
-  end
-
-  def print_person(person, index = 1)
-    "#{index + 1}) [#{person.class.name}] ID: #{person.id}, Name: #{person.name}, Age: #{person.age}"
+    PrintBook.new.print_all(@books)
+    nil
   end
```

Comment:
- [ ] The sole responsibility of the `main.rb` file should simply be to start up and request initial user inputs. This method definition should be in another file. Kindly fix this.